docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,269 @@
+
+*Query IOCs* are those with a ``match_type`` of ``query``; their ``values_list`` contains a single string that
+specifies a query compatible with process searches.  For example, the following IOC looks for the process ``git.exe``
+that does *not* connect to one of a specified list of IP addresses:
+
+::
+
+    {
+        "id": "example_1",
+        "match_type": "query",
+        "values": ["process_name:git.exe NOT (netconn_ipv4:35.158.151.206 OR netconn_ipv4:1.1.244.78
+                    OR netconn_ipv4:80.18.61.229 OR netconn_ipv4:80.18.61.228)"]
+    }
+
+Query IOCs must always use field-prefixed queries (key-value pairs); they do not support just searching for a value
+without a field specified.  Values in query clauses that do not specify fields will be ignored.
+
+:Wrong: ``process_name:chrome.exe AND 192.168.1.1``
+:Right: ``process_name:chrome.exe AND netconn_ipv4:192.168.1.1``
+
+Query IOCs may search on CIDR address ranges, e.g.: ``netconn_ipv4:192.168.0.0/16``.
+
+Query IOCs are searched every 5 minutes by the Carbon Black Cloud, and are tested against a rolling window of the
+last hour's worth of data for the organization.  (They will *not* generate hits or alerts for process attributes that
+were reported more than an hour in the past.)  They may employ any searchable field as documented
+`here <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/platform-search-fields/>`_,
+and may employ complex query logic.
+
+*Ingress IOCs* are those with a ``match_type`` of ``equality`` or ``regex``; they use the ``field`` element to specify
+the name of a field to examine the value of, and the ``values_list`` element to specify a list of values to match
+against (in the case of ``match_type`` being ``equality``) or regular expressions to match against (in the case of
+``match_type`` being ``regex``).  For example, this IOC will match any process that initiates a connection to one of
+two listed IP addresses:
+
+::
+
+    {
+        "id": "example_2",
+        "match_type": "equality",
+        "field": "netconn_ipv4",
+        "values": ["8.8.8.8", "1.160.120.15"]
+    }
+
+This IOC will match any process running with an executable name beginning with "quake":
+
+::
+
+    {
+        "id": "example_3",
+        "match_type": "regex",
+        "field": "process_name",
+        "values": ["quake.*\\.exe"]
+    }
+
+(Note the use of the backslash to escape the '.' that separates the file extension from the name.  It must be doubled
+to escape it in Python itself.)
+
+Ingress IOCs are searched as soon as the data is received from any endpoint, and may use any process field
+(as documented
+`here <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/platform-search-fields/>`_;
+the fields tagged with ``PROCESS``)
+in their ``field`` element, whether searchable or not.  For the searches they are capable of, they are more efficient
```

Comment:
Lines 169-172 are a little bit confusing. I can see there is an in-line link, but what comes after the semi-colon doesn't seem like a full sentence. I could be missing something though.

Reply:
I put in a few more words so it looks more complete.